sigs.yaml

```
@@ -1933,12 +1933,12 @@ sigs:
   label: network
   leadership:
     chairs:
-    - github: caseydavenport
-      name: Casey Davenport
-      company: Tigera
     - github: dcbw
       name: Dan Williams
       company: Red Hat
+    - github: shaneutt
+      name: Shane Utt
+      company: Kong
     - github: thockin
       name: Tim Hockin
       company: Google
```

Optional suggestion, if you want to add yourself to the emeritus leads section
```suggestion
      company: Google
    emeritus_leads:
    - github: caseydavenport
      name: Casey Davenport
```